tutorials/grails3-send-push-notification-using-firebase-cloud-messaging

```diff
@@ -0,0 +1,206 @@
+---
+title: Grails 3 Sending Push Notification using Google Firebase Cloud Messaging
+description: Step by step tutorial on how to sending push notification using Grails 3 and Google Firebase Cloud Messaging
+author: didinj
+tags: Firebase, Firebase Cloud Messaging, FCM, Grails, Grails 3, Push Notification
+date_published: 2017-08-23
+---
+
+This tutorial shows how to send push notification using [Firebase Cloud Messaging (FCM)](https://firebase.google.com/docs/cloud-messaging) and [Grails 3 Framework](https://grails.org/) Web Application.
+
+## Objectives
+
+- Configure Firebase to send push notification via Web Application
+- Create a new Grails 3.3.0 Web Application
+- Create function for sending push notification on Grails 3.3.0
+
+## Before You Begin
+
+The following is required for this tutorial:
+
+- Terminal or Command Line
+- Text Editor or IDE ([Atom](https://atom.io/), [Netbeans](https://netbeans.org/))
+- [Java Development Kit 8](http://www.oracle.com/technetwork/java/javase/downloads/jdk8-downloads-2133151.html)
+- Grails 3.3.0 SDK (https://grails.org/download.html)
+- Google or Gmail account for accessing Firebase dashboard
+
+## Costs
+
+This tutorial uses Firebase Cloud Messaging service which is free for limited features and capacities.
+
+## Configure Firebase Cloud Messaging
+
+1. Open your browser then go to [Firebase](https://firebase.google.com/).
+2. Login using your Gmail account then click Go to Console button.
+3. Click add project button then give it a name. For this tutorial name it "GrailsFCM" then click Create Project button.
+4. It will redirected to Project dashboard. Click the gear icon in the left menu and select Project Settings.
```

Comment:
change "It will redirected to Project dashboard" to "It will redirect to the Project dashboard"